### PR TITLE
Add Shell command to clear console

### PR DESCRIPTION
**Pull request by mauricema**
This patch set added Shell CLS command to clear screen.  As part of it, it also implement clear screen for GFX console.

### CODE REVIEW

File: BootloaderCommonPkg/Library/GraphicsLib/GraphicsLib.c [hunk 1]
```diff
@@ -306,6 +319,23 @@ FrameBufferWrite (
     return 0;
   }
 
+  // Handle clear screen
+  if ((NumberOfBytes == 4) && (!CompareMem (ANSI_ESCAPE_SEQ_CLEAR_SCREEN, Buffer, 4))) {
```
Comment (gdong1) [inline BootloaderCommonPkg/Library/GraphicsLib/GraphicsLib.c]: the return of CompareMem is INTN. should compare with 0 by EDK2 coding style.

File: BootloaderCommonPkg/Library/GraphicsLib/GraphicsLib.c [hunk 2]
```diff
@@ -306,6 +319,23 @@ FrameBufferWrite (
...
+    // Clear screen
+    SetMem (Console->TextDisplayBuf, Console->Rows * Console->Cols, 0);
+    // Zero framebuffer
+    GfxInfoHob = Console->GfxInfoHob;
+    Length = (GfxInfoHob->GraphicsMode.HorizontalResolution * GfxInfoHob->GraphicsMode.PixelsPerScanLine) * 4;
+    SetMem64 (
+        (UINT32 *) (UINTN)(GfxInfoHob->FrameBufferBase),
+        Length,
+        0
+      );
```
Comment (gdong1) [inline BootloaderCommonPkg/Library/GraphicsLib/GraphicsLib.c]: For this simple function and parameter is not very long, better use single line?
SetMem64 ((UINT32 *) (UINTN)(GfxInfoHob->FrameBufferBase), Length, 0);

Comment (mauricema) [reply BootloaderCommonPkg/Library/GraphicsLib/GraphicsLib.c]: Sure, will do it.

File: BootloaderCommonPkg/Library/GraphicsLib/GraphicsLib.c (hunk 1 above)
Comment (mauricema) [reply BootloaderCommonPkg/Library/GraphicsLib/GraphicsLib.c]: Will change it.

File: BootloaderCommonPkg/Library/GraphicsLib/GraphicsLib.c (hunk 2 above)
Comment (mauricema) [reply BootloaderCommonPkg/Library/GraphicsLib/GraphicsLib.c]: Updated per comments.

**Pull request merged**